<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminHome</class>
 <widget class="QMainWindow" name="AdminHome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Main window background */
QMainWindow {
    background-color: #f5f5f5; /* Светло-серый фон */
}

/* General button style */
QPushButton {
    background-color: #4CAF50; /* Зеленый фон */
    border: none; /* Без границы */
    color: white; /* Белый текст */
    padding: 10px 24px; /* Внутренний отступ */
    text-align: center; /* Выравнивание текста по центру */
    font-size: 16px; /* Размер шрифта */
    margin: 4px 2px; /* Внешний отступ */
    border-radius: 12px; /* Закругленные углы */
}

QPushButton:hover {
    background-color: #45a049; /* Темно-зеленый фон при наведении */
}

QPushButton:pressed {
    background-color: #398439; /* Ещё темнее при нажатии */
}

/* Line edit fields */
QLineEdit {
    border: 1px solid #ccc; /* Светло-серая граница */
    border-radius: 10px; /* Закругленные углы */
    padding: 10px; /* Внутренний отступ */
    font-size: 16px; /* Размер шрифта */
    background-color: #fff; /* Белый фон */
    margin: 4px 0; /* Внешний отступ */
}

/* Line edit focus style */
QLineEdit:focus {
    border: 1px solid #4CAF50; /* Зеленая граница при фокусе */
}

/* Labels */
QLabel {
    font-size: 16px; /* Размер шрифта */
    color: #333; /* Темно-серый текст */
}

/* Message box buttons */
QMessageBox QPushButton {
    background-color: #4CAF50; /* Зеленый фон */
    color: white; /* Белый текст */
    border: none; /* Без границы */
    padding: 10px 20px; /* Внутренний отступ */
    border-radius: 5px; /* Закругленные углы */
}

QMessageBox QPushButton:hover {
    background-color: #45a049; /* Темно-зеленый фон при наведении */
}

QMessageBox QPushButton:pressed {
    background-color: #398439; /* Ещё темнее при нажатии */
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>711</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Добавить пользователя</string>
     </attribute>
     <widget class="QPushButton" name="RegisterButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>280</y>
        <width>321</width>
        <height>57</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>57</height>
       </size>
      </property>
      <property name="text">
       <string>Зарегестрировать</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>301</width>
        <height>305</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>67</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Имя</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameLine"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>67</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Фамилия</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="surnameLine"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Отчество</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="patronymicLine"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>67</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Пароль</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="passwordline"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>53</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>67</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Роль</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>43</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Администратор</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Студент</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Преподаватель</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>40</y>
        <width>321</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>61</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Адресс</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="addressLine"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Телефон</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="phoneLine"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>61</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Группа</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="groupLine"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Введите основные данные:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>321</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Введите дополнительные данные:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Удалить пользователя</string>
     </attribute>
     <widget class="QPushButton" name="deleteButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>100</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Введите ФИО:</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>251</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="minimumSize">
            <size>
             <width>67</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Имя</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="deleteNameLine"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="minimumSize">
            <size>
             <width>67</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Фамилия</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="deleteSurnameLine"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Отчество</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="deletePatronymicLine"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Добавить факультатив</string>
     </attribute>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Введите информацию о факультативе:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Кто ведет:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addFacultativ">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>260</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>40</y>
        <width>271</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <widget class="QLabel" name="label_22">
           <property name="minimumSize">
            <size>
             <width>67</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>67</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Имя</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="NamePrepodavatilEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <item>
          <widget class="QLabel" name="label_23">
           <property name="minimumSize">
            <size>
             <width>67</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>67</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Фамилия</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="SurnamePrepodavatilEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_19">
         <item>
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>Отчество</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="PatronumicPrepodavatilEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>346</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Название факультатива</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="NameFacultativeEdit">
           <property name="minimumSize">
            <size>
             <width>165</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Вид проведения</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_2">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>43</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>165</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Практика</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Лекция</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Лабараторная работа</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Время проведения</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTimeEdit" name="timeFacultativeEdit">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>43</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>165</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Количество мест</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="SeatsFacultativeEdit">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Группа</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="GroupFacultativeEdit">
           <property name="maximumSize">
            <size>
             <width>260</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>711</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Выйти из сессии</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
